configure will pick up gtk-doc even though we pass --disable-gtk-doc and if
junked by dpb(1) we will error out with:
aclocal-1.15: error: aclocal: file 'm4/gtk-doc.m4' does not exist

Index: autogen.sh
--- autogen.sh.orig
+++ autogen.sh
@@ -10,7 +10,7 @@ test -z "$srcdir" && srcdir=.
     exit 1
 }
 
-GTKDOCIZE=$(which gtkdocize 2>/dev/null)
+#GTKDOCIZE=$(which gtkdocize 2>/dev/null)
 if test -z $GTKDOCIZE; then
         echo "You don't have gtk-doc installed, and thus won't be able to generate the documentation."
         rm -f gtk-doc.make
